app-src/set-up-parent-frame-communications.js

```diff
@@ -21,7 +22,11 @@ function setUpParentFrameCommunications(document) {
 }
 
 function playVideos() {
-  paella.player.videoContainer.play();
+  $(document).trigger(paella.events.play);
```

Comment:
The triggering of the play event notifies the VideoContainer to start playing, and also notifies plugs such as the PlayPauseButtonPlugin. It is the normal way a plugin communicates the play request. 

I'm not sure why Jim chose to call the play function directly, but calling it does not trigger a play event and prevents other UI components from knowing that the video is playing.